Wikipedia/Code/DiffHeaderExtendedView.swift

```diff
@@ -0,0 +1,117 @@
+
+import UIKit
+
+protocol DiffHeaderActionDelegate: class {
+    func tappedUsername(username: String)
+    func tappedRevision(revisionID: Int)
+}
+
+class DiffHeaderExtendedView: UIView {
+ 
+    @IBOutlet var contentView: UIView!
+    @IBOutlet var stackView: UIStackView!
+    @IBOutlet var summaryView: DiffHeaderSummaryView!
+    @IBOutlet var editorView: DiffHeaderEditorView!
+    @IBOutlet var compareView: DiffHeaderCompareView!
+    @IBOutlet var divViews: [UIView]!
+    @IBOutlet var editorDivView: UIView!
+    @IBOutlet var compareDivView: UIView!
+    
+    private var viewModel: DiffHeaderViewModel?
+    
+    var delegate: DiffHeaderActionDelegate? {
```

Comment:
needs `weak`

Reply:
Fixed - since this was a computed property and just passes the assignment into its children I thought maybe it wasn't necessary. But I also thought I would get a compile failure after adding weak but I'm not seeing that now 🤷‍♀️